src/CMakeLists.txt

```diff
@@ -598,3 +599,8 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 set_property(TARGET ncnn PROPERTY FOLDER "libncnn")
 set_property(TARGET ncnn-generate-spirv PROPERTY FOLDER "libncnn")
 
+# simplemath
+if(NCNN_SIMPLEMATH)
+    add_definitions(-DNCNN_SIMPLEMATH)
+    message(STATUS "NCNN_SIMPLEMATH enabled")
+endif()
```

Comment:
add NCNN_SIMPLEMATH definition in platform.h.in
cmake will define it for you on needed

Reply:
Done.